drivers/Makefile.dep

```diff
@@ -81,16 +81,18 @@ ifneq (,$(filter bm%280,$(USEMODULE)))
   USEMODULE += bmx280
 endif
 
-ifneq (,$(filter cc110x,$(USEMODULE)))
-  USEMODULE += ieee802154
+ifneq (,$(filter cc110%,$(USEMODULE)))
+  USEMODULE += cc110x
+  USEMODULE += cc1xxx_common
   USEMODULE += luid
+  USEMODULE += netif
   USEMODULE += xtimer
-  ifneq (,$(filter gnrc_netdev_default,$(USEMODULE)))
-    USEMODULE += gnrc_cc110x
-  endif
   FEATURES_REQUIRED += periph_gpio
   FEATURES_REQUIRED += periph_gpio_irq
   FEATURES_REQUIRED += periph_spi
+  ifneq (,$(filter gnrc_ipv6,$(USEMODULE)))
+    USEMODULE += ieee802154
```

Comment:
Raw means that there the packet has no L2 header, I don't think that is what you meant.

Reply:
Then lets keep the question more generic: Why is this needed for IPv6? Which parts of `ieee802154` are reused here?

Reply:
It uses the 6LoWPAN adaption layer that allows providing a 1280Byte IPv6 MTU even when the layer 2 maximum PDU is way lower. It basically fragments the IPv6 packets (bigger then the layer 2 PDU) and reassembles them on the receiver site.

@miri64: Would it be enough to add `gnrc_sixlowpan` as module instead only? That would be certainly more readable.

Reply:
I have to look into your code in detail (I'm just looking at the comment view atm), but it *should* work with just `gnrc_sixlowpan`, otherwise our modularization is pretty broken.

Reply:
OK, thanks! I'll try this and do some testing. If it works, I'll push a fixup.

Reply:
It works :-)